backend/src/main/java/atlantbh/restaurants/models/Category.java

```diff
@@ -0,0 +1,54 @@
+package atlantbh.restaurants.models;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+import javax.persistence.Entity;
+import javax.persistence.OneToMany;
+import java.util.Collection;
+
+@Entity
+public class Category extends BaseModel<Category> {
+
+    private String name;
+    private Collection<Restaurant> restaurants;
+
+    public Category(String category) {
+        this.name = category;
+    }
+
+    public Category() {
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    @OneToMany(mappedBy = "category")
+    @JsonIgnore
+    public Collection<Restaurant> getRestaurants() {
+        return restaurants;
+    }
+
+    public void setRestaurants(Collection<Restaurant> restaurants) {
+        this.restaurants = restaurants;
+    }
+
+
+    @Override
+    public Category duplicate(Category model) {
+        Category category = new Category();
+        category.setName(model.getName());
+        category.setRestaurants(model.getRestaurants());
+        return category;
+    }
+
+    @Override
+    public void update(Category data) {
+        if (data.getName() != null) setName(data.getName());
```

Comment:
Use {} always please. Also, null check is not needed here, since name and restaurants are not required on this model? If they are, add validation annotation to getters setters..

Reply:
Fixed in 8c841a20e489a31c66d177c8682339f9d4f19de0